src/BackgroundLayerPreview/BackgroundLayerPreview.tsx

```diff
@@ -110,8 +127,8 @@ export const BackgroundLayerPreview: React.FC<BackgroundLayerPreviewProps> = ({
       return;
     }
 
-    const newBgLayer = mainMap?.getLayerGroup().getLayers().getArray()
-      .find(l => getUid(l) === layerId);
+    const newBgLayer = MapUtil.getAllLayers(mainMap)
+      .find(l => getUid(l) === layerId) as any;
```

Comment:
Can typing be more specific here? eg. `OlLayer` or `OlLayerBase`?

Reply:
Right, for some reason I had that in there, but it's not necessary. Removed.